backend/devices/bitbox02/keystore.go

```diff
@@ -51,15 +51,14 @@ func (keystore *keystore) RootFingerprint() ([]byte, error) {
 	return keystore.device.RootFingerprint()
 }
 
-// SupportsAccount implements keystore.Keystore.
-func (keystore *keystore) SupportsAccount(coin coin.Coin, meta interface{}) bool {
+// SupportsCoin implements keystore.Keystore.
+func (keystore *keystore) SupportsCoin(coin coin.Coin) bool {
```

Comment:
> We want to be able to list the coins available for a keystore in a dropdown

then it feels like a simpler way is to just return a slice of all supported coins?

I saw a new `Backend.AvailableCoins` which does return a slice. So, why not make this keystore method also return a slice?

```go
func (k *keystore) SupportedCoins() []coin.Coin
```

Reply:
Also possible. `AvailableCoins()` still needs to exist to filter testnet/regtest. Do you have a strong preference to change it?

Reply:
I have many preferences :)
It just felt natural to return a slice. Up to you.

But I've realized, I'd actually name Backend.AvailableCoins `SupportedCoins`. See the other comment in the frontend API.

Reply:
done